packages/reports/src/app.tsx

```diff
@@ -1,11 +1,18 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import Routes from 'Containers/routes.jsx';
 import { MobxContentProvider } from 'Stores/connect';
 import 'Sass/app.scss';
 import initStore from './init-store'; // eslint-disable-line import/extensions
+import RootStore from './Stores/index';
```

Comment:
@akmal-deriv You can use the [TStores](https://github.com/binary-com/deriv-app/blob/master/packages/stores/src/useStore.tsx#L5) type from `@deriv/stores` package instead and extend it with local stores type like this:

```ts
import type { TStores } from '@deriv/stores';
import type ProfitTableStore from './Profit/profit-store';
import type StatementStore from './Statement/statement-store';

export type TRootStore = TStores & {
    modules: {
        profit_table: ProfitTableStore;
        statement: StatementStore;
    };
};
```

Reply:
@shayan-deriv @niloofar-deriv Just for your information, You guys can install and use the `TStores` type from `@deriv/stores` while migrating to TS, You don’t have to use the package itself as that would be done in refactoring but for now you can use `TStores` type and extend and improve it in other packages instead of having the root store type as any. 

For example, if we are using `root_store.client.foo` in some component and the type of `foo` is `any` we can add the type in [this](https://github.com/binary-com/deriv-app/blob/master/packages/stores/types.ts) file and make it accessible for all other packages as well instead of defining root store type in each package over and over.

Not sure who else is working on TS migration now and needs to be inform about this, I just leave it to you guys 😅🙏🏻

Reply:
Thank u @farzin-deriv for sharing this, we will take care of this :smile: